python/desc/imsim/cartesian_zernikes.py

```diff
@@ -0,0 +1,445 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""Defines the zernike polynomials in cartesian coordinates using the Noll
+indexing scheme.
+"""
+
+from numpy import sqrt
+
+
+def z_1(x, y):
```

Comment:
Yes, in the standard convention it should be one. Fortunately, any overall normalization coefficients get filtered out by the fitting process. Out of curiosity, how large was the profiling difference?

Reply:
Yeah, I agree that as long as this code is self-contained inside imSim it doesn't matter (none of the normalizations matter in that case).  If we were to ever want to compare to another double Zernike calculation though, then it's important to get this right.

As for the profiling, the difference depends on the number of (x,y) pairs being evaluated, but I get galsim.zernike as between 1.5x and 3x faster than cartesian_zernikes.gen_superposition().  I don't think this is likely to be the tall pole though.  This suggestion is mostly to avoid code duplication.  Since we're already importing galsim, I think it makes sense to use the code that's there.  The added speed and flexibility are nice bonus though.